client/bin/smee.js

```diff
@@ -7,11 +7,17 @@ const Client = require('..')
 program
   .usage('[options]')
   .option('-u, --url <url>', 'URL of the webhook proxy service. Default: https://smee.io/new')
+  .option('-t, --target <target>', 'URL of the target service the events will forwarded to. Default: http://127.0.0.1:port/path')
```

Comment:
Given the implementation, we might want to be a little bit more explicit here:

```suggestion
  .option('-t, --target <target>', 'Full URL (including protocol and path) of the target service the events will forwarded to. Default: http://127.0.0.1:PORT/PATH')
```